tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -755,29 +755,29 @@ def test_send_status_to_db(self, cluster):
 
     @pytest.mark.level("minimal")
     @pytest.mark.clustertest
-    def test_status_scheduler_basic_flow(self, cluster):
+    def test_status_scheduler_basic_flow(self, ondemand_aws_https_cluster_with_auth):
```

Comment:
any reason this is a specific cluster fixture?

Reply:
nope, it was used for my local testing, reverted it to use a generic cluster fixture.